adobe_vipm/adobe/client.py

```diff
@@ -1063,9 +1061,31 @@ def get_customer_deployments(
         response.raise_for_status()
         return response.json()
 
+    def get_customer_deployments_by_status(
```

Comment:
I would say that it should be a util function that either filters out inactive deployments or splits them by active/nonactive. The thing is that if you put it to the client it should be a http query filter in that case. But since it is important to log what was removed here, I think better to have an util function that splits deployments by status